spec/fixtures/pattern_search/test_paths/salus.yaml

```diff
@@ -0,0 +1,10 @@
+scanner_configs:
+  PatternSearch:
+    matches:
+      - regex: hello
+        forbidden: true
+        exclude_filepaths:
```

Comment:
can you exclude filepaths and directories at the same time? If so, can you make a test case for it?

Reply:
Yes, just added test case